src/artifactSearcher.ts

```diff
@@ -8,7 +8,12 @@ import { registerCommand } from "./extension";
 import { executeJavaLanguageServerCommand } from "./jdtls/commands";
 import { applyWorkspaceEdit } from "./utils/editUtils";
 
-const unresolvedCode: string[] = ["16777218", "570425394"];
+// Please refer to https://help.eclipse.org/2019-06/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Fconstant-values.html
+// tslint:disable-next-line: variable-name
```

Comment:
follow tslint rule to rename it as `UNDEFINED_TYPE` etc

Reply:
or create a enum `CategorizedProblem`, e.g. 
```
enum CategorizedProblem {
UNDEFINED_TYPE = "xxxx",
...
}
```

Reply:
I thought using the same name as that of eclipse document would be better. seems not to be what I thought ahhh.